### ISSUE TITLE
Tabs?

### ISSUE THREAD

**Issue by moll** [dmnd/dedent#10]
Hey,

Is it me or doesn't `dedent` support dedenting tabs?

Cheers

**Comment by dmnd** [comment]
I don't use tabs, but PRs welcome. Though I wonder what the expected behaviour is when tabs and spaces are mixed.

**Comment by moll** [comment]
Presumably ignoring whichever _wasn't_ at the start of the line would be sane behavior. :)

**Comment by adrianjost** [comment]
Hi, I ran into [the issue](https://github.com/pocka/storybook-addon-vue-info/issues/79) that this library still isn't supporting tabs. I did some research with the result that packages like [outdent](https://www.npmjs.com/package/outdent#spaces-and-tabs) are handling tabs like a single space character. This makes the implementation really simple. In most cases there are no mixed spaces/tabs and if there are and we would handle tabs like spaces, we have an equally broken result as we would encounter with the current implementation (since spaces are trimmed, but tabs are still there). 

I will try to implement this and open a pull request, if you are still fine with that @dmnd.
